packages/source-airtable/index.js

```diff
@@ -1,28 +1,80 @@
 const Airtable = require('airtable')
+// eslint-disable-next-line node/no-unpublished-require
+const { deprecate } = require('../../gridsome/lib/utils/deprecate')
```

Comment:
```suggestion
const { deprecate } = require('gridsome/lib/utils/deprecate')
```
The `gridsome` package is in another location when this plugin is installed in a project. And adding `gridsome` as a peer dependency should get rid of the ESlint message.

Reply:
Yes, this is the line that I was curious how to solve. Thanks!